StructureDefinition/NHSDigital-Patient-PDS.StructureDefinition.xml

```diff
@@ -38,28 +46,13 @@
       <path value="Patient.address" />
       <min value="1" />
     </element>
-    <element id="Patient.generalPractitioner">
-      <path value="Patient.generalPractitioner" />
-      <min value="1" />
-      <max value="1" />
-    </element>
-    <element id="Patient.generalPractitioner.identifier">
-      <path value="Patient.generalPractitioner.identifier" />
-      <min value="1" />
-    </element>
-    <element id="Patient.generalPractitioner.identifier.system">
-      <path value="Patient.generalPractitioner.identifier.system" />
-      <min value="1" />
+    <element id="Patient.address.extension:addressKey">
+      <path value="Patient.address.extension" />
+      <sliceName value="addressKey" />
     </element>
-    <element id="Patient.generalPractitioner.identifier.value">
-      <path value="Patient.generalPractitioner.identifier.value" />
-      <min value="1" />
-    </element>
-    <element id="Patient.generalPractitioner:registeredGPPractice">
-      <path value="Patient.generalPractitioner" />
-      <sliceName value="registeredGPPractice" />
-      <min value="1" />
-      <max value="1" />
+    <element id="Patient.address.extension:addressKey.extension">
+      <path value="Patient.address.extension.extension" />
+      <min value="2" />
```

Comment:
Could we remove the constraint on addressKey entirely? There may be up to 2 for PDS there may also be none at all.